Example/Info.plist

```diff
@@ -15,7 +15,7 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>$(MARKETING_VERSION)</string>
+	<string>$(MARKETING_VERSION)$(RMA_DEMO_APP_BUILD_TYPE)</string>
```

Comment:
we have to be aware that we can't change the version number directly from Xcode target version field anymore

<img width="740" alt="image" src="https://user-images.githubusercontent.com/1507391/104891187-f2a74a80-59b3-11eb-9131-ef02c1db8eba.png">